docs/conf.py

```diff
@@ -0,0 +1,408 @@
+# html_theme_path = []
+
+# The name for this set of Sphinx documents.  If None, it defaults to
+# "<project> v<release> documentation".
+# html_title = None
+
+# A shorter title for the navigation bar.  Default is the same as html_title.
+# html_short_title = None
+
+# The name of an image file (relative to this directory) to place at the top
+# of the sidebar.
+# html_logo = None
+
+# The name of an image file (within the static path) to use as favicon of the
+# docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
+# pixels large.
+# html_favicon = None
+
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ["_static"]
+html_extra_path = ["_html"]
+
+# If not '', a 'Last updated on:' timestamp is inserted at every page bottom,
+# using the given strftime format.
+# html_last_updated_fmt = '%b %d, %Y'
+
+# If true, SmartyPants will be used to convert quotes and dashes to
+# typographically correct entities.
+# html_use_smartypants = True
+
+# Custom sidebar templates, maps document names to template names.
+# html_sidebars = {}
+
+# Additional templates that should be rendered to pages, maps page names to
+# template names.
+# html_additional_pages = {}
+
+# If false, no module index is generated.
+# html_domain_indices = True
+
+# If false, no index is generated.
+# html_use_index = True
+
+# If true, the index is split into individual pages for each letter.
+# html_split_index = False
+
+# If true, links to the reST sources are added to the pages.
+# html_show_sourcelink = True
+
+# If true, "Created using Sphinx" is shown in the HTML footer. Default is True.
+# html_show_sphinx = True
+
+# If true, "(C) Copyright ..." is shown in the HTML footer. Default is True.
+# html_show_copyright = True
+
+# If true, an OpenSearch description file will be output, and all pages will
+# contain a <link> tag referring to it.  The value of this option must be the
+# base URL from which the finished HTML is served.
+# html_use_opensearch = ''
+
+# This is the file name suffix for HTML files (e.g. ".xhtml").
+# html_file_suffix = None
+
+# Output file base name for HTML help builder.
+htmlhelp_basename = "DataRobotAirflowDoc"
+
+
+# -- Options for Markdown output -----------------------------------------------
+# Adapted from https://github.com/liran-funaro/sphinx-markdown-builder?tab=readme-ov-file#configurations
+
+# If set to True, then anchors will be added before each section/function/class signature.
+# This allows references to a specific anchor in the document.
+# markdown_anchor_sections: True
+# markdown_anchor_signatures: True
+
+# If set to True, adds metadata to the top of each document containing author, copyright,
+# and version.
+markdown_docinfo: True
```

Comment:
Hey 👋 

So while working on an (unsolicited) PR for mypy updates - [seen here](https://github.com/datarobot/airflow-provider-datarobot/pull/109/) - I now see a mypy error that _could_ signify this should be an assignment like so:
`markdown_docinfo = True`

Since I see the error:
```
docs/conf.py:281:19: error: Invalid type: try using Literal[True] instead?  [valid-type]
    markdown_docinfo: True
```

So mypy wants the typing to be updated which makes me think that maybe the syntax is off here - even though the comments above _do_ suggest using a colon.

Anyway, just wanted to share - _hopefully_ helpful 🤷